docs/docs/deploying-to-s3-cloudfront.md

```diff
@@ -54,6 +54,23 @@ And finally, add the deployment script to your `package.json`:
 That's it!
 Run `npm run build && npm run deploy` to do a build and have it immediately deployed to S3!
 
+## Deploying with `.env` variables
+
+Some deployments require passing environment variables. To deploy with environment variables, update the deployment script to your `package.json`:
+
+```js
+"scripts" : {
+    ...
+    "deploy": "npm run -n \"-r dotenv/config\" && npm run build && gatsby-plugin-s3 deploy"
+}
+```
+
+This command requires running `dotenv` first before deploying to s3. Additionally, if you have multiple AWS profiles in your machine, you can deploy by declaring your `AWS_PROFILE` before the deploy script:
```

Comment:
It would help to describe what exactly is happening in this script. I also don't see references to dotenv in this doc yet. Do we need to instruct users to install it before they can run this?

Reply:
Cool! It would be worth mentioning that so that people know. Can you add a note about it?